### DISCUSSION TITLE
Airflow task is getting killed due to /tmp size

### DISCUSSION THREAD

**Discussion by prasadbestha** [apache/airflow#21588] [Q&A]
Hi,

We have a data pipeline which is running fine in aws datapipe service. we are trying to deploy the same code into Airflow and execute.
Airflow task is killed when sql query trigged, the select query out put should be around 6gb. We have tried by changing node small to medium and worker nodes 5 from 2, Still facing the same issue.

error: **/tmp/airflowtmp4jxqv8sa/xxxxxuvvijdjd: line 38:  2988 Killed**  

how to handle this situation , do we have any property to increase /tmp size.

Thanks,
Prasad

**Comment by potiuk** [comment]
It's only question of your deployment. There is nothing in Airlfow to determine the size of /tmp folder.
